inc/functions.php

```diff
@@ -5033,6 +5017,43 @@ function build_theme_select($name, $selected="", $tid=0, $depth="", $usergroup_o
 }
 
 /**
+ * Get the theme data of a theme id.
+ *
+ * @param int The theme id of the theme.
+ * @return array The theme data
+ */
+function get_theme($tid)
+{
+	global $tcache, $db;
+
+	if(!is_array($tcache))
+	{
+		$query = $db->simple_select('themes', 'tid, name, pid, allowedgroups', "pid!='0'", array('order_by' => 'pid, name'));
```

Comment:
Well, doesn't matter if you copied it or not - it doesn't make sense <b>here</b>. In theme selector sorting them is not surprising (they're displayed), but here it doesn't do anything.